### PR TITLE
Add security context to installer in helm config

### PR DESCRIPTION
**Pull request by maroshii**

### CODE REVIEW

File: src/content/self-hosted/helm-configuration.mdx [hunk 1]
```diff
@@ -437,6 +437,7 @@ The jobs that deploy your [development environments from Git](development/deploy
 - `sshSecretName`: The name of the secret that contains the private key used when cloning git repos using ssh. If it doesn't exist, the key and the secret will be automatically generated by Okteto.
 - `resources`: The resources for pods created by the installer jobs.
 - `priorityClassName`: The priority class for pods created by the installer job pods. The PriorityClass must already exist in your cluster before using this setting. This value has precedence over `globals.priorityClassName` if both are set. If this value is not set, the pods will inherit the priority class defined by the value set in `globals.priorityClassName`.
+- `securityContext`: The security context for the installer job container. It's not set by default, and it follow the [same syntax used in Kubernetes](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)
```
Comment (pchico83) [inline src/content/self-hosted/helm-configuration.mdx:440]:
```suggestion
- `securityContext`: The security context for the installer job container. It's not set by default, and it follows the [same syntax used in Kubernetes](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)
```

**Review by pchico83 — APPROVED** [review]
We shouldn't merge this one until 1.25 is out

**Comment by ifbyol** [comment]
> We shouldn't merge this one until 1.25 is out

Why? this is included in the `1.25` release

**Comment by pchico83** [comment]
> > We shouldn't merge this one until 1.25 is out
> 
> Why? this is included in the `1.25` release

@ifbyol @maroshii my bad, ready to merge then. I got confused with the PR for the docker secret warning

**Comment by ifbyol** [comment]
> > > We shouldn't merge this one until 1.25 is out
> > 
> > 
> > Why? this is included in the `1.25` release
> 
> @ifbyol @maroshii my bad, ready to merge then. I got confused with the PR for the docker secret warning

Ok ok, yeah, a few things going on at the same time and it's easy to be confused. All good then 👍

**Review by ifbyol — APPROVED** [review]

**Pull request merged**